Components/Grid/grid.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Components/Grid/grid.css">
+    <script src="https://kit.fontawesome.com/d53bb09bcf.js" crossorigin="anonymous"></script>
+
+    <title>Grid page</title>
+</head>
+
+<body>
+    <div class="Container parentContainer" id="modalDrawerMainBody">
+
+        <header>
+            <div class="navBar space-between">
+                <div class="nav-left space-evenly">
+                    <i class="fas fa-bars hamburgerMenu" id="hamburgerMenuModal"></i>
+                    <h3 class="m-r-4">Modal Drawer</h3>
+                </div>
+                <div class="nav-right space-evenly">
+                    <h3 class="m-r-4"><a href="/index.html">Get Started</a></h3>
+                    <h3 class="m-r-4"><a href="/home.html">Documentation</a></h3>
```

Comment:
what's `m-r-4`? Maybe try using  self-descriptive classnames

Reply:
For margin, padding, font-weight I have used variables like m-4 for margin: 4rem, p-1 for padding: 1rem etc.